tests/integration/targets/elasticache/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Integration testing for the elasticache module
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+  collections:
+    - amazon.aws
+  block:
```

Comment:
Can you add an `always` block please, which will always delete elasticache instances in case of errors.